src/appleseed.shaders/src/maya/as_maya_anisotropic.osl

```diff
@@ -269,7 +269,7 @@ shader as_maya_anisotropic
                 );
         }
 
-        if (transparency)
+        if (max(transparency) > 0.0)
```

Comment:
@est77 same applies, why max() ? The inequality test will suffice since transparency is in [0,1] range, any component != 0 would trigger the clause.